hand_off_target_upsert.go

```diff
@@ -0,0 +1,28 @@
+package client
+
+import (
+	"context"
+	"net/http"
+)
+
+type UpsertHandOffTargetParams struct {
+	// ID is your identifier of choice for this hand-off target.
```

Comment:
Is this validated at all? I wonder if we should validate it (in the backend), and perhaps also provide an example

Reply:
It's validated on the server — https://github.com/gradientlabs-ai/wearegradient/blob/8530c9f0f83594f0214a7120736618456f202dd2/apis/public-api/hand_off_target_upsert.go#L29. I'll add some additional context about what is valid [like we do in the api-docs](https://github.com/gradientlabs-ai/api-docs/blob/cae2c7e7ced47aa49f9fd1a27e1d70c267af51c7/source/index.html.md?plain=1#L175).